impress-org/givewp
# Reports for "All Time" are now inclusive of the first day.

Resolves #5399

## Description



Both the start date and the end date were being appended with the same time string: `24:00:00`. I assume that this was intended only for the end date to be inclusive of the end of the day, but was also being set for the start date, which excluded the day's donations.

## Affects



Updates the `sanitizeDate` callback for Report Endpoints.

## Visuals



![image](https://user-images.githubusercontent.com/10858303/96925030-8b93fb80-1481-11eb-8ca2-61bce83a4b0c.png)

First day's donations are included in the "All Time" report data.

![image](https://user-images.githubusercontent.com/10858303/96924903-54254f00-1481-11eb-83e9-6494734175e9.png)


## Pre-review Checklist




## Testing Instructions



1. Create a donation
1. Go to Donations -> Reports
1. Click "All Time"

The first day's donations should be included in the report data.
